docs/guides/formulasC2/build-template.md

```diff
@@ -0,0 +1,260 @@
+
+{% include tip.html content=" If you plan to use the formula instance that you create while testing only for testing, consider appending the name test to it. " %}
+
+To test a formula template:
+
+1. At the top of the formula visualization, click **Setup Test**.
+![Setup Test](img/set-up-test.png)
+2. Click **Select Instance**.
+3. Either choose an existing formula instance (and skip down to the Select Trigger step) or click **Add New Instance**.
+  4. Enter a name for the new formula instance.
+  5. For each variable, click the variable and add an element instance or enter a value.
+  6. Click **Create Instance**.
+  7. Select the instance that you just created.
+8. Click **Select Trigger**.
+9. Provide trigger information, depending on the type of trigger.
+
+## Create Variables
+
+You can create two types of variables: Element Instance variables that will be replaced by element instances and Value variables that will be replaced by values entered at the formula instance. When you create a variable, note the Formula Step Variable Name which is how you refer to the variable in the formula context. See [Formula Variables](reference.html/#formula-variables) for more about variables in the formula context.
+
+To create a formula variable:
+
+1. Open the formula template. On the Formulas page, hover over the Formula Card, and then click **Open**.
+5. Click **Variables**.
+![Variables](img/variables.png)
+2. Select the type of variable to create.
+3. Enter a name for the variable. You can include spaces, but spaces are removed to create the Formula Step Variable Name, which is how the variable is referred to in the formula context.
+4. Click **Save**.
+
+## Edit and Delete Variables
+
+You can edit the name of a variable or remove a variable from a formula template.
+
+{% include note.html content="You cannot delete a formula variable if an instance of the formula exists.  " %}
+
+To edit or delete a formula variable:
+
+1. Open the formula template. On the Formulas page, hover over the Formula Card, and then click Open.
+5. Click **Variables**.
+![Variables](img/variables.png)
+2. Click **Edit Variable**.
+3. Select the variable to edit or delete, and then:
+  * Edit the name, and then click **Save**. Or,
+  * Click **Delete**.
+
+## Review Executions
+
+You can troubleshoot formulas by reviewing executions. The executions are split into three columns from left to right:
+
+* Formula Executions &mdash; the list of executions of the selected formula template. Failed executions are clearly marked.
+* Steps &mdash;The steps in the formula and an indication of their success <img src="img/icon-step-success.png" alt="Success" class="inlineImage"> or failure <img src="img/icon-step-failure.png" alt="Failure" class="inlineImage">.
+* values &mdash;The details associated with the selected step.
```

Comment:
Can we say "Execution Values: - the request and response details associated with the selected step.